src/ts_catalog/compression_chunk_size.c

```diff
@@ -43,3 +46,75 @@ ts_compression_chunk_size_delete(int32 uncompressed_chunk_id)
 
 	return count;
 }
+
+TSDLLEXPORT bool
+ts_compression_chunk_size_get(int32 chunk_id, Form_compression_chunk_size form)
+{
+	ScanIterator iterator =
+		ts_scan_iterator_create(COMPRESSION_CHUNK_SIZE, AccessExclusiveLock, CurrentMemoryContext);
+	bool found = false;
+
+	Assert(form != NULL);
+
+	init_scan_by_uncompressed_chunk_id(&iterator, chunk_id);
+	ts_scanner_foreach(&iterator)
+	{
+		TupleInfo *ti = ts_scan_iterator_tuple_info(&iterator);
+		bool should_free;
+		HeapTuple tuple = ts_scanner_fetch_heap_tuple(ti, false, &should_free);
+		memcpy(form, GETSTRUCT(tuple), sizeof(FormData_compression_chunk_size));
```

Comment:
Nit: Will avoid problems if the type of `form` changes.
```suggestion
		memcpy(form, GETSTRUCT(tuple), sizeof(*form));
```